config.js

```diff
@@ -269,6 +269,10 @@ var config = {
     // Enable lock room for all moderators, even when userRolesBasedOnToken is enabled and participants are guests.
     // lockRoomGuestEnabled: false,
 
+    // When enabled the password used for locking a room is restricted to up to the number of digits specified
+    // roomPasswordNumberOfDigits: 10,
+    // default: roomPasswordNumberOfDigits: false,
```

Comment:
Typo?

Reply:
What typo?

Reply:
I guess there is no typo. The double colons threw me off. I don't think the other config values follow this format for explaining the default but I don't know if it really matters much.